release-team/role-handbooks/communications/README.md

```diff
@@ -13,18 +13,19 @@ Additionally, there are specific deliverables that must come from the release pr
 ### Skills and Experience Required
 
 - Strong written and verbal communications skills
-- Experience with the Kubernetes release process enough to understand how communications milestones fit into the overall release
-- Existing relationships with the CNCF, relevant media personnel and outlets, Kubernetes project managers, and vendor stakeholders
 - A working knowledge of Kubernetes concepts
+- Fundamental GitHub skills, experience with open source projects
+- Experience with the Kubernetes release process enough to understand how communications milestones fit into the overall release
 - Project management experience is helpful
+- Existing relationships with the CNCF, relevant media personnel and outlets, Kubernetes project managers, and vendor stakeholders is helpful
 
 ### Expected Time Investment
 
-The Kubernetes release cycle usually spans 12 weeks. The typical workload for the communications team is _very_ light the first few weeks. In the later weeks of the release cycle, the workload can get very heavy.
+The Kubernetes release cycle spans 16 weeks, though may run longer. The typical workload for the communications team is very light during the first few weeks. In the later weeks of the release cycle, the workload can become heavy, and often work can continue on a few weeks after release.
```

Comment:
```suggestion
The Kubernetes release cycle spans 15 weeks, though may run longer. The typical workload for the communications team is very light during the first few weeks. In the later weeks of the release cycle, the workload can become heavy, and often work can continue on a few weeks after release.
```
We state 15 weeks in the [release cadence kep](https://github.com/kubernetes/enhancements/tree/master/keps/sig-release/2572-release-cadence#summary), the end of the year release tends to be 16 weeks